model_analyzer/config/input/config_command_profile.py

```diff
@@ -178,6 +178,13 @@ def _fill_config(self):
                 default_value=DEFAULT_DURATION_SECONDS,
                 description=
                 'Specifies how long (seconds) to gather server-only metrics'))
+        self._add_config(
```

Comment:
I think that this option exposes too much of the internals of model analyzer. Ideally, we do not want the user to know about the CPU monitor. We want them to only deal in the level of CPU metrics. What if we moved the metric tags from the MetricsManager and made it user configurable with a reasonable default set of metrics (no CPU metrics). Then in the metrics manager, we use that option to determine whether we need to run the CPU monitor or not. @Tabrizian would like your input on this as well.

Reply:
Agreed. I think using the metric tags to determine whether the CPU monitor should be enabled or not is a good idea. Having this option can complicate things. For example, if the CPU monitor is disabled and cpu metrics are in the metrics tag what should the MA do?

Reply:
I see, so ultimately the CPU monitor follows the metric tags, but the auto mode logic should be used to display warning(s) if there is a discrepancy between what is specified and what is supposed to do.

Reply:
@aramesh7 @Tabrizian Please confirm if this setup is what you are expecting. 
I will be working on the analyze part for this change.